packages/react-components/react-progress/stories/ProgressBar/ProgressBarThickness.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { makeStyles, shorthands } from '@fluentui/react-components';
 import { ProgressBar } from '@fluentui/react-progress';
+import { Field } from '@fluentui/react-field';
```

Comment:
_nit:_ Alphabetically order import packages

```suggestion
import { Field } from '@fluentui/react-field';
import { ProgressBar, ProgressBarProps } from '@fluentui/react-progress';
```